### DISCUSSION TITLE
useSeoMeta is not updating the title of ogTags

### DISCUSSION THREAD

**Discussion by dbzx10299** [nuxt/nuxt#20813] [Questions]
I have the following code in a file called `[...slug].vue`,

What happens is when I navigate to a different blogPost, the `title` and `description` update but not `ogTitle` and `ogDescription`.  They stay the same until the page is refreshed. I am not sure if I implemented something wrong or if this is a bug, it is a bit strange the title and description tags are the only ones that update on navigation.

```js
<script setup>
const { data: blogPost } = await useAsyncData(`content-${path}`, () => {
  return queryContent().where({ _path: path }).findOne()
})

useSeoMeta({
  title: blogPost.value.title,
  description: blogPost.value.description,
  ogTitle: blogPost.value.title,
  ogDescription: blogPost.value.description
})
</script>
```

**Comment by riddla** [comment]
Maybe you need to use the composable in a "reactive" fashion as shown over at https://unhead.harlanzw.com/integrations/vue/reactivity#reactivity?

```typescript
useSeoMeta({
  title: () => blogPost.value.title,
  description: () => blogPost.value.description,
  ogTitle: () => blogPost.value.title,
  ogDescription: () => blogPost.value.description
})
```
